parsec/core/gui/tr/parsec_fr.po

```diff
@@ -673,14 +673,15 @@ msgstr "utilisateur@hote.com"
 msgid "ACTION_USER_INVITE_DO_INVITE"
 msgstr "Inviter l'utilisateur"
 
-msgid "TEXT_USER_INVITE_CANCEL_INVITE_QUESTION_TITLE"
-msgstr "Annuler l'invitation"
-
 msgid "TEXT_USER_INVITE_CANCEL_INVITE_QUESTION_CONTENT"
 msgstr ""
 "Êtes-vous sûr de vouloir annuler cette invitation ? Le lien d'invitation "
 "qu'a reçu l'utilisateur par email deviendra invalide."
 
+msgid "TEXT_USER_INVITE_SUCCESS_email"
+msgstr ""
+"L'invitation à rejoindre votre organisation a été envoyé avec succès à l'adresse : <b>\"{email}\"</b>."
```

Comment:
"EnvoyéE".
In both French and English I would simply remove the colon and the quotes around the email.

Reply:
My bad ^^